src/components/result-form.js

```diff
@@ -55,6 +56,14 @@ const ResultForm = ({ user, criterion, max }) => {
     [user, criterion, dirty, lockAcquired]
   );
 
+  if (readOnly) {
+    return (
+      <div className='position-relative'>
+        {value}
+      </div>
+    );
```

Comment:
Relative position was used for status-notice, it is not necessary here.

```suggestion
    return <div>{value}</div>;
```

Reply:
removed